source/js/main.js

```diff
@@ -7,7 +7,6 @@ import Analytics from "./analytics.js";
 
 import JoinUs from "./components/join/join.jsx";
 import Petition from "./components/petition/petition.jsx";
-import People from "./components/people/people.jsx";
```

Comment:
People.jsx relies on `person.jsx`, so if that's now orphaned, we should remove that, too.